src/jit/lsra.cpp

```diff
@@ -4564,6 +4564,8 @@ void LinearScan::setFrameType()
         }
     }
 
+    frameType = FT_EBP_FRAME;
```

Comment:
I'd prefer if we filed added a comment re: this workaround and checked this bit of the change in on its own.